apps/api/src/lib/itemUtils.ts

```
@@ -0,0 +1,81 @@
+
+  displayValue = actualValue
+
+  const stars = getStarsFromRollValue(inputValue, raw <= 0);
+  return {
+    stars,
+    percentage: rollPercentage,
+    displayValue
+  };
+}
+
+export function isCost(key: string): Boolean {
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)